lib/fluent/plugin/out_uri_decode.rb

```diff
@@ -1,16 +1,16 @@
-class Fluent::URIDecoder < Fluent::Output
+require 'fluent/plugin/output'
+require 'uri'
+
+class Fluent::Plugin::URIDecoderOutput < Fluent::Plugin::Output
   Fluent::Plugin.register_output('uri_decode', self)
 
-  config_param :tag, :string, :default => nil
-  config_param :remove_prefix, :string, :default => nil
-  config_param :add_prefix, :string, :default => nil
-  config_param :key_name, :string, :default => nil
-  config_param :key_names, :string, :default => ''
+  helpers :event_emitter
 
-  def initialize
-    super
-    require 'uri'
-  end
+  config_param :tag, :string, default: nil
+  config_param :remove_prefix, :string, default: nil
```

Comment:
I think these configurations should be removed in next version, because these configurations are not worked well.

I've checked logic. `tag_mangle` method says:

* `tag` is exclusive with `add_prefix`
* `tag` is exclusive with `remove_prefix`
* `add_prefix` and `remove_prefix` are not exclusive with each other

But in `configure`, following case will raise error:

* specify all `tag`, `add_prefix` and `remove_prefix`
* omit all `tag`, `add_prefix` and `remove_prefix`